simulariumio/data_objects/simularium_file_data.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, Union
+from .trajectory_data import TrajectoryData
+from .frame_data import FrameData
+
+
+class SimulariumFileData:
+    def __init__(self, file_contents: Union[str, bytes]):
```

Comment:
maybe the base class doesn't need to have the file_contents even as a constructor argument?